### DISCUSSION TITLE
why dont any of the default annoyances filters remove youtube search results garbage/junk ?

### DISCUSSION THREAD

**Discussion by goodstuff626** [uBlockOrigin/uBlock-issues#3506] [Q&A]
I figured they should be part of the default easylist Annoyances, adguard Annoyances, ublock Annoyances, but none of them block the garbage/junk youtube search results - 'people also search for' From related searches' channels new to you' recommendations'.
What would be the best filter list I can use for always removing/hiding the youtube search results garbage junk?

**Comment by MasterKia** [comment]
See:
https://www.reddit.com/r/uBlockOrigin/wiki/solutions/youtube/

Other than that, search `filterlists.com` or use the [element picker](https://github.com/gorhill/ublock/wiki/Element-picker) make your own filters.

**Comment by Yuki2718** [comment]
Because what is garbage for you can be what is needed for others.

**Reply by freezer2022** [reply]
By the way, it reminds a similar request with a similar reply (hence a duplicate of):
- https://github.com/uBlockOrigin/uAssets/discussions/24263
